api/app/utils/review.inference.js

```diff
@@ -88,9 +88,9 @@ async function calculateWordDiff(data) {
       modifiedText = modifiedText.join(" ");
     }
     if (
-      (originalText.trim().startsWith("[") &&
-        originalText.trim().endsWith("]")) ||
```

Comment:
Splitting the if block should fix the issue. That is we account for both cases rather than an OR (||)